_sass/web.scss

```diff
@@ -90,10 +90,9 @@ $masthead-border-color: $color-light !default;
 
 // Website sidebar/navigation
 $nav-bar-text-size: 0.8em !default;
-$nav-bar-hide: true !default;
-$nav-bar-prompt: "Contents" !default;
```

Comment:
As mentioned in earlier comment, I'm curious to know your thinking behind removing this variable as a way to insert and control the nav prompt.